src/crypto.js

```diff
@@ -0,0 +1,45 @@
+const ethereumjsUtil = require("ethereumjs-util");
+
+/*
+* FUNCTIONS TAKEN FROM OPEN_ATTESTATION FRAMEWORK
```

Comment:
Add openattestation utils to its package.json's files and import them instead

https://stackoverflow.com/questions/33016714/how-to-require-a-file-other-than-the-npm-main-file-in-node